tools/testkit/src/fevm.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![allow(dead_code)]
+
+use fvm::executor::{ApplyKind, ApplyRet, Executor};
+use fvm_ipld_encoding::tuple::*;
+use fvm_ipld_encoding::{strict_bytes, BytesSer, RawBytes};
+use fvm_shared::address::Address;
+use fvm_shared::message::Message;
+use fvm_shared::{ActorID, METHOD_CONSTRUCTOR};
+
+use crate::{Account, BasicTester};
+
+pub fn create_contract(tester: &mut BasicTester, owner: &mut Account, contract: &[u8]) -> ApplyRet {
+    let create_msg = Message {
+        from: owner.account.1,
+        to: Address::new_id(10), // EAM
+        gas_limit: 10_000_000_000,
+        method_num: EAMMethod::CreateExternal as u64,
+        params: RawBytes::serialize(BytesSer(contract)).unwrap(),
+        sequence: owner.seqno,
+        ..Message::default()
+    };
+    let create_mlen = create_msg.params.len();
+
+    let create_res = tester
+        .executor
+        .as_mut()
+        .unwrap()
```

Comment:
```suggestion
        .expect("executor not initialized")
```

Reply:
i think we still need the unwrap, no?

Reply:
Expect is the same thing, it just gives a nicer error message.